doc/authoring_command_modules/README.md

```diff
@@ -199,28 +199,36 @@ azdev style --module <module> [--pylint] [--pep8]
 Submitting Pull Requests
 ------------------------
 
-### Modify Change Log
+### Format PR Title
 
-Modify the `HISTORY.rst` for any customer-facing changes. If a module has changed at all since a previous release so that a version bump is required, it is fine to add a generic entry that says "* Minor fixes.".
+History notes are auto-generated based on commit messages and descriptions starting from [S165](https://github.com/Azure/azure-cli/milestone/82). The commmit message for the squashed merge commit defaults to the PR title. Starting from 01/30/2020, we require all the PR titles to follow the below format:
```

Comment:
not related to this PR, what's our plan for PR created before 1/30/2020, but not merged when 1/30? we'll update all the title?

Reply:
For thoses PRs, if they modified HISTORY.rst, they would have conflicts with latest dev branch. We can just remove the notes in HISTORY.rst and change PR title or description accordingly. 

If the PR does not modify HISTORY.rst, we should change those start with [] to {}. We can leave the rest that do not follow our new format untouched since they will be ignored when generating history notes.

We can publicize such action during the release meeting on 1/30/2020.